documentation/sphinx/source/configuration.rst

```diff
@@ -387,6 +387,8 @@ FoundationDB will never use processes on the same machine for the replication of
 
     FoundationDB replicates data to three machines, and at least three available machines are required to make progress. This is the recommended mode for a cluster of five or more machines in a single datacenter.
 
+    .. note:: When running in cloud environments with managed disks that are already replicated and persistent, ``double`` replication may still be considered for 5+ machine clusters.  This will result in a lower total read throughput, but offers a reasonable tradeoff for cost.
```

Comment:
Does this come with availability risks that are worth mentioning? A loss of two machines could make you unavailable, and you have less budget for doing planned activities that bring down a machine, as doing so would leave you with no fault tolerance.

Reply:
Is it feasible to re-attach a given EBS volume to a machine with a different IP address and expect it to "just work", or is there more fiddling required?

What if the machine was a coordinator? I realize that isn't directly related to double replication, but it is related to healing a cluster running on EBS after a failure.

Reply:
For storage/log, it would "just work".  For coordinator, you'd need to distribute a new cluster file to point to the new IP address, but otherwise it would work.

I'll add the availability and maintenance concern in addition to the read throughput callout.